Issue with release 1.2

There seems to be an issue with release 1.2.

I've installed it like so

```
npm install -g sql-language-server
```

I'm getting this error when JupyterLab communicates with the sql-language-server
```
 [ERROR] default - uncaughtException TypeError: server_1.TextDocuments is not a constructor
    at createServerWithConnection (/opt/conda/lib/node_modules/sql-language-server/dist/src/createServer.js:48:21)
    at createServer (/opt/conda/lib/node_modules/sql-language-server/dist/src/createServer.js:311:12)
    at Object.handler (/opt/conda/lib/node_modules/sql-language-server/dist/bin/cli.js:25:37)
    at Object.runCommand (/opt/conda/lib/node_modules/sql-language-server/node_modules/yargs/lib/command.js:238:44)
    at Object.parseArgs [as _parseArgs] (/opt/conda/lib/node_modules/sql-language-server/node_modules/yargs/yargs.js:1063:30)
    at Object.get [as argv] (/opt/conda/lib/node_modules/sql-language-server/node_modules/yargs/yargs.js:1004:21)
    at Object.<anonymous> (/opt/conda/lib/node_modules/sql-language-server/dist/bin/cli.js:30:5)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
```

I checked the version of vscode-languageserver using

```
npm list -g --depth 2 
```
it is at version vscode-languageserver@8.0.0-next.8

this version is different than the one specified in the source code which is vscode-languageserver@8.0.0-next.3

And when I look at the vscode-languageserver@8.0.0-next.8 version the file /opt/conda/lib/node_modules/sql-language-server/node_modules/vscode-languageserver/lib/common/server.js 

does not contain a `TextDocuments` class

there seems to be a version issue with the npm package. I'm not sure how to resolve this issue.

--- a/.gitignore
+++ b/.gitignore
@@ -14,3 +14,4 @@ packages/server/dist/*
 !.yarn/sdks
 !.yarn/versions
 .pnp.*
+.node-version

--- a/packages/server/package.json
+++ b/packages/server/package.json
@@ -50,7 +50,7 @@
     "sqlint": "^1.2.0",
     "sqlite3": "^5.0.2",
     "vscode-languageclient": "^6.1.3",
-    "vscode-languageserver": "^8.0.0-next.3",
+    "vscode-languageserver": "8.0.0-next.8",
     "vscode-languageserver-protocol": "^3.15.3",
     "vscode-languageserver-textdocument": "^1.0.1",
     "yargs": "^17.3.1"

--- a/packages/server/src/createServer.ts
+++ b/packages/server/src/createServer.ts
@@ -4,7 +4,7 @@ import {
   CompletionItem,
   CompletionParams,
 } from 'vscode-languageserver/node'
-import { TextDocuments } from 'vscode-languageserver/lib/common/server'
+import { LSPObject, TextDocuments } from 'vscode-languageserver'
 import { CompletionTriggerKind } from 'vscode-languageserver-protocol/lib/common/protocol'
 import { TextDocument } from 'vscode-languageserver-textdocument'
 import { CodeAction, TextDocumentEdit, TextEdit, Position, CodeActionKind } from 'vscode-languageserver-types'
@@ -12,7 +12,7 @@ import cache from './cache'
 import { complete } from './complete'
 import createDiagnostics from './createDiagnostics'
 import createConnection from './createConnection'
-import SettingStore from './SettingStore'
+import SettingStore, { Connection as SettingConnection } from './SettingStore'
 import { Schema } from './database_libs/AbstractClient'
 import getDatabaseClient from './database_libs/getDatabaseClient'
 import initializeLogging from './initializeLogging'
@@ -186,13 +186,19 @@ export function createServerWithConnection(connection: Connection, debug = false
     if (!hasConfigurationCapability) {
       return
     }
-    const connections = change.settings?.sqlLanguageServer?.connections ?? []
+    if (!Object.prototype.hasOwnProperty.call(change.settings, 'sqlLanguageServer')) {
+      logger.debug('onDidChangeConfiguration', "it doesn't have sqlLanguageServer property")
+      return 
+    }
+    const sqlLanguageServerSetting = ((change.settings as LSPObject).sqlLanguageServer) as LSPObject
+
+    const connections = (sqlLanguageServerSetting.connections ?? []) as SettingConnection[]
     if (connections.length > 0) {
       SettingStore.getInstance().setSettingFromWorkspaceConfig(connections)
     }
 
     // On configuration changes we retrieve the lint config
-    const lint = change.settings?.sqlLanguageServer?.lint
+    const lint = ((sqlLanguageServerSetting).lint) as RawConfig
     lintConfig = lint
     if (lint?.rules) {
       documents.all().forEach(v => {
@@ -272,7 +278,7 @@ export function createServerWithConnection(connection: Connection, debug = false
     ) {
       try {
         SettingStore.getInstance().changeConnection(
-          request.arguments && request.arguments[0] || ''
+          request.arguments && request.arguments[0]?.toString() || ''
         )
       } catch (e) {
         const err = e as NodeJS.ErrnoException
@@ -291,7 +297,7 @@ export function createServerWithConnection(connection: Connection, debug = false
         })
         return
       }
-      const document = documents.get(uri)
+      const document = documents.get(uri.toString())
       const text = document?.getText()
       if (!text) {
         logger.debug('Failed to get text')
@@ -305,7 +311,7 @@ export function createServerWithConnection(connection: Connection, debug = false
       logger.debug('Fix all fixable problems', text, result[0].fixedText)
       connection.workspace.applyEdit({
         documentChanges: [
-          TextDocumentEdit.create({ uri, version: document!.version }, [
+          TextDocumentEdit.create({ uri: uri.toString(), version: document!.version }, [
             TextEdit.replace({
               start: Position.create(0, 0),
               end: Position.create(Number.MAX_VALUE, Number.MAX_VALUE)

